src/main/java/gtPlusPlus/core/util/player/PlayerCache.java

```diff
@@ -19,17 +19,13 @@ public class PlayerCache {
 
     public static void initCache() {
         if (CORE.PlayerCache == null) {
-            try {
-
-                if (cache != null) {
-                    CORE.PlayerCache = PlayerCache.readPropertiesFileAsMap();
-                    Logger.INFO("Loaded PlayerCache.dat");
-                }
-
-            } catch (final Exception e) {
-                Logger.INFO("Failed to initialise PlayerCache.dat");
+            if (cache.exists()) {
+                CORE.PlayerCache = PlayerCache.readPropertiesFileAsMap();
+                Logger.INFO("Loaded PlayerCache.dat");
+            }
+            if (CORE.PlayerCache == null) {
```

Comment:
maybe just use `else`?

Reply:
I tried else but that would just be if it doesn’t exist. This “catches” it existing but resulting in a null return when parsing.